torch/testing/_internal/common_device_type.py

```diff
@@ -722,12 +697,38 @@ def split_if_not_empty(x: str):
     env_only_for = split_if_not_empty(os.getenv(PYTORCH_TESTING_DEVICE_ONLY_FOR_KEY, ''))
     env_except_for = split_if_not_empty(os.getenv(PYTORCH_TESTING_DEVICE_EXCEPT_FOR_KEY, ''))
 
-    desired_device_type_test_bases = filter_desired_device_types(desired_device_type_test_bases,
-                                                                 env_except_for, env_only_for)
+    return filter_desired_device_types(desired_device_type_test_bases, env_except_for, env_only_for)
+
 
 
+# Adds 'instantiated' device-specific test cases to the given scope.
```

Comment:
Any particular reason for moving this function? Doesn't look like there's any functional changes here.